torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -252,10 +258,35 @@ def render(self, kernel: ROCmTemplateKernel, op: "CKGemmOperation", **kwargs) ->
         template_buffer_node = kwargs.get("template_buffer_node", None)
         if template_buffer_node is not None:
             self.output_node = template_buffer_node
-        instance_definition, instance_type = self.emit_ck_instance(op)
         X, W = self.input_nodes[0], self.input_nodes[1]
         Y = self.output_node
-        Bias = None  # TBD support gemm_bias
+        Bias = self.input_nodes[2] if 3 == len(self.input_nodes) else None
+
+        op = copy.deepcopy(op)
+
+        op.c_shuffle_block_transfer_scalar_per_vector_n_per_block = (
```

Comment:
The op object is the result of parsing a CK template without multiple D, and then is used to render a template with multiple D. Most of the parameters are the same, but this one needs to be adjusted. First element of the tuple relates to gemm result, and second element of the tuple relates to bias (D). In case of several biases fused together, it will contain one element for C as well as each of the Ds.